WordPress/src/debug/AndroidManifest.xml

```diff
@@ -7,10 +7,14 @@
     <application
         tools:replace="android:name,android:supportsRtl"
         android:name=".WordPressDebug"
-        android:supportsRtl="true"
-        />
+        android:supportsRtl="true">
```

Comment:
⚠️ I think we should be adding this to `main/AndroidManifest.xml` instead of `debug/AndroidManifest.xml`. Sorry, I couldn't test this since I can't seem to build any of the release variants.

Reply:
Very good point, I forgot to move this to the release Manifest.

Note: I'm only able to build release variants by removing the sentry plugin and the minifier, I didn't investigate the issue much, will ping platform9.

Reply:
Fixed in 406688b6d3058c0307c15e63dd006a9bc31e3576